content/blog/posts/o3de-graphics-atom-intro.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Exploring the world of graphics via Atom, O3DE’s renderer
+date: 2022-06-23
+slug: graphics-atom-intro
+author: Judy Ng
+blog_img: ""
+full_img: ""
+---
+Hello all! I’m Judy, and I’m an intern for the Open 3D Engine (O3DE) Atom team. I interned with Atom in the summer of 2021 and came back to the team to intern again this summer 2022! So far this summer, I’ve learned graphics fundamentals and applied that knowledge to write a custom shader and make my own material type. Here’s some details about my journey learning about graphics and Atom.
```

Comment:
Suggested rewrite: Hello all! I'm Judy, an intern at Amazon working on the Open 3D Engine (O3DE), specifically the Atom renderer. I interned with Amazon in the summer of 2021, focused on contributing to Atom, and have returned to Amazon as an intern again this summer! So far, ...